src/build/mod.rs

```diff
@@ -8,6 +8,8 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+//! Running builds for the client.
```

Comment:
To be clear, here, these are builds the RLS has decided to do, not builds the user has explicitly requested in the client (which are handled purely by the client).